Validation/RecoTrack/plugins/TrackingNtuple.cc

```diff
@@ -1190,6 +1191,7 @@ class TrackingNtuple : public edm::one::EDAnalyzer<edm::one::SharedResources> {
   // (first) index runs through hits
   DetIdAll simhit_detId;
   DetIdAllPhase2 simhit_detId_phase2;
+  std::vector<TrackerGeometry::ModuleType> simhit_moduleType;
```

Comment:
please do not introduce a dependency on CMSSW format in the ntuple.
Simply using an int would be good enough.

Reply:
You're right, I have removed the dependency by casting this to int.